src/main/java/diet/dietsession/CommandLib.java

```diff
@@ -0,0 +1,28 @@
+package diet.dietsession;
+
+import diet.dietsession.command.*;
+
+import java.util.Hashtable;
+
+public class CommandLib {
```

Comment:
Since it's the same we can refactor the two to a shared package then use it for all parts?

Reply:
Agreed, but I think that it would be better after all the functions are up after the first iteration.